feature-governance-api/src/main/java/io/novafoundation/nova/feature_governance_api/domain/referendum/vote/ConstructAccountVoteUtils.kt

```diff
@@ -0,0 +1,25 @@
+package io.novafoundation.nova.feature_governance_api.domain.referendum.vote
+
+import io.novafoundation.nova.feature_governance_api.data.network.blockhain.model.AccountVote
+import io.novafoundation.nova.feature_governance_api.data.network.blockhain.model.VoteType
+import io.novafoundation.nova.runtime.multiNetwork.runtime.types.custom.vote.Conviction
+import io.novafoundation.nova.runtime.multiNetwork.runtime.types.custom.vote.Vote
+import java.math.BigInteger
+
+fun constructAccountVote(amount: BigInteger, conviction: Conviction, voteType: VoteType): AccountVote {
```

Comment:
I suggest to have this in `compation` object of `AccountVote` rather a standalone function present somewhere separately